build.sbt

```diff
@@ -5,28 +5,18 @@ import wartremover.WartRemover.autoImport.wartremoverExcluded
 
 val appName: String = "payments-email-verification"
 
-ThisBuild / scalaVersion  := "2.13.12"
+ThisBuild / scalaVersion  := "3.3.4"
 ThisBuild / majorVersion  := 3
```

Comment:
could we bump up the major version to `4`? It'd mark a major shift in the codebase with the intro of scala 3